go.mod

```diff
@@ -1,68 +1,73 @@
 module github.com/awslabs/volume-modifier-for-k8s
 
-go 1.19
+go 1.21
 
 require (
 	github.com/golang/protobuf v1.5.3
-	github.com/google/go-cmp v0.5.9
-	github.com/kubernetes-csi/csi-lib-utils v0.13.0
-	google.golang.org/grpc v1.56.3
-	google.golang.org/protobuf v1.30.0
-	k8s.io/api v0.27.1
-	k8s.io/apimachinery v0.27.1
-	k8s.io/client-go v0.27.1
-	k8s.io/csi-translation-lib v0.27.1
-	k8s.io/klog/v2 v2.90.1
-	k8s.io/kubectl v0.27.1
+	github.com/google/go-cmp v0.6.0
+	github.com/kubernetes-csi/csi-lib-utils v0.17.0
+	google.golang.org/grpc v1.60.1
```

Comment:
Possibly dumb question, but I just want to make sure that we aren't affected by the [same google.golang.org/grpc v1.59 regression the csi sidecars faced (#153)](https://github.com/kubernetes-csi/csi-lib-utils/pull/153) where "gRCP connections are closed after 30 minutes" and caused sidecar restarts. 

It seems that in [volume-modifier-for-k8s/pkg/client/csi_client.go](https://github.com/awslabs/volume-modifier-for-k8s/blob/80541378b1babf4ec61448cc8e9844f3cc7f221a/pkg/client/csi_client.go#L74) we rely on [ClientConn.Connect](https://pkg.go.dev/google.golang.org/grpc#ClientConn.Connect), instead of [ClientConn.Dial](https://pkg.go.dev/google.golang.org/grpc#Dial), so therefore we cannot even pass [withIdleTimeout(0)](https://pkg.go.dev/google.golang.org/grpc#WithIdleTimeout), so perhaps we're safe? 

I can test for this next release by idling the driver for 40 min and checking if the sidecar still works afterwards.

Reply:
IIUC this sidecar should not be affected by that regression.

> I can test for this next release by idling the driver for 40 min and checking if the sidecar still works afterwards.

👍